src/models/layerMetadata/layer3DMetadata.ts

```diff
@@ -361,7 +363,7 @@ export class Layer3DMetadata implements ILayer3DMetadata, IMetadataCommonModel {
   })
   @fieldConfig({
     category: FieldCategory.MAIN,
-    infoMsgCode: ['info-field-tooltip.minResolutionMeter.min', 'info-field-tooltip.minResolutionMeter.max'],
+    infoMsgCode: ['info-field-tooltip.meter.tooltip', 'info-field-tooltip.minResolutionMeter.min', 'info-field-tooltip.minResolutionMeter.max'],
```

Comment:
What do you mean by 'info-field-tooltip.meter.tooltip'?

Reply:
i didn't changed it - i will pull from master again, to make sure i'm alligned with it